api/pkg/apis/v1alpha1/managers/stage/stage-manager.go

```diff
@@ -631,6 +631,7 @@ func (s *StageManager) HandleTriggerEvent(ctx context.Context, campaign model.Ca
 
 		waitGroup.Wait()
 		close(results)
+		// gofail: var afterProvider string
```

Comment:
Please also add another line "DO NOT REMOVE THIS COMMENT" as you did in other files

Reply:
fixed